Makefile

```diff
@@ -5,8 +5,8 @@ default: build
 init-examples:
 	@echo "==> Creating symlinks for example/ projects to terraform-provider-castai binary"; \
 	TF_PROVIDER_FILENAME=terraform-provider-castai; \
-	GOOS=`go tool dist env | awk -F'=' '/^GOOS/ { print $$2}' | tr -d '"'`; \
-	GOARCH=`go tool dist env | awk -F'=' '/^GOARCH/ { print $$2}' | tr -d '"'`; \
+	GOOS=`go tool dist env | awk -F'=' '/^GOOS/ { print $$2}' | tr -d '";'`; \
```

Comment:
Why do we need this change?

Reply:
in my local environment `go tool dist env` provides list with quotes and semicolons. Like this:
`
GOHOSTARCH="arm64";
GOHOSTOS="darwin";
`

Reply:
So this change remove semicolons from the name too